### PR TITLE
move parts from https://github.com/oda-hub/doc-multi-user/

### PR DESCRIPTION
**Pull request by ferrigno**
I added references in interface.md to the other documents defining user fields and roles.
I think it is relevant to understand how we want to manage users. The possibility of multiple sources of identification could carry a degree of issues with the interface definition.

### CODE REVIEW

File: interfaces.md
Comment (volodymyrss) [inline interfaces.md:18]: this link will vanish too. plan-roles-user.md already exists, we can refer to that already

File: interfaces.md
Comment (volodymyrss) [inline interfaces.md:13]: I would say users are  defined, though generated is fine if you prefer.
The reference to PR branch will go away, so maybe we can directly use that.

**Review by volodymyrss — CHANGES_REQUESTED** [review]

**Comment by volodymyrss** [comment]
could you please not merge things if review is incpmplete

**Comment by volodymyrss** [comment]
if you have happy with your update you  can re-request review there is a button there near review.

**Comment by ferrigno** [comment]
Just made a mistake on using github interface, sorry.

**Comment by volodymyrss** [comment]
review is not just or even necessarily for approval, is for keeping track. unreviewed changes are disruptive, since they will not be appreciated, and will introduce confusion.
even if in this case it is clear that the is no misunderstanding, let's follow a practice?

**Comment by ferrigno** [comment]
I agree that awareness of mutual contributions is fundamental, but it is just not clear to me at which point something can be closed and merged, I thought that implementing your comments was sufficient to "close with comment". However, it looks that this did not actually merged the branch, which, by the way, should be deleted after merging, I think. I leave the final action to you.

**Comment by volodymyrss** [comment]
> I agree that awareness of mutual contributions is fundamental, but it is just not clear to me at which point something can be closed and merged, I thought that implementing your comments was sufficient to "close with comment". However, it looks that this did not actually merged the branch, which, by the way, should be deleted after merging, I think. I leave the final action to you.

ok, it's a technical aspect. at the same top right corner where you put reviewers, you can "re-request" review, there is button. 
then they will be converted to pending (mine is currently in "changes requested" and @burnout87 's is "pending").

it also says in big red letters just above here "Changes requested".

once both reviews are approched, they convert to green tick marks, and other comments become all green.
then we can merge. I'd say anyone can merge at that point, but usually it's natural it's the original submitter. 
in this case, we can complete ourselves, since really the content is small. but it's just to agree on the workflow (in this case it is standard github PR workflow)

**Review by volodymyrss — APPROVED** [review]

**Comment by volodymyrss** [comment]
I made non-negligible changes to this, but I can not assign @ferrigno to review since he created it. So I wait for @burnout87  review or further comments from @ferrigno and merge. Further changes are possible after.

File: interfaces.md
Comment (burnout87) [inline interfaces.md:25]: Good, I like this explanation, very clear.

**Comment by volodymyrss** [comment]
Oh, sorry @burnout87 turns out your review was not approved. The comments indicated so, so I merged, my bad. 
Is it ok? Else we try to revert.

**Comment by burnout87** [comment]
It's ok no problem. It was more of a curiosity for me. Things that will anyway become clear soon

**Comment by volodymyrss** [comment]
> It's ok no problem. It was more of a curiosity for me. Things that will anyway become clear soon

it's also so that you acknowledge you are aware of these changes and this doc. It would be a shame if you missed it!

**Comment by burnout87** [comment]
Indeed, but I saw them all and commented where I wantend to

**Pull request merged**
